data/features/m/m.sh

```diff
@@ -21,7 +21,7 @@ m() {
     for arg in "${@:2}"; do
       # search for lines that start with the second argument ($2) preceded by one or more spaces (^ +)
       # The -A5 option tells grep to also output five lines of context after each match.
-      man "$1" | grep -hnI -E "^ +${arg}" -B 5 -A 5 --color='auto'
+      man "$1" | grep -hnI -E "+${arg}" -B 5 -A 5 --color='auto'
```

Comment:
If I remove the + sign from the condition, the function stops working as I said you.

Reply:
Change it, it is doing nothing.

Reply:
As it is:
![imagen](https://user-images.githubusercontent.com/14998495/232472140-55beb326-fc8b-4084-80e5-4edc332c28e5.png)

Without the '+' sign:
![imagen](https://user-images.githubusercontent.com/14998495/232472315-d035a6f7-34cd-446a-8514-e8c68a155d93.png)

Reply:
> That is such a bad example since `-` has special meaning in bash and needs to be escaped.

If you ignore my comments the revision process stops.

Delete the `+`. `m grep -E` is a bad example, the dash is interpreted as special character. Can you try with a regular example? Or try it  scaping the special character.  Show proof of what you are doing.